ansible/roles/cyhy_reporter/tasks/main.yml

```diff
@@ -61,10 +61,10 @@
 - name: Create cron job for daily CyHy notifications
   cron:
     name: "Generate and send daily CyHy notifications"
-    minute: 0
-    hour: 10
+    minute: '0'
+    hour: '10'
     user: cyhy
-    job: cd /var/cyhy/notifications && ./create_send_notifications.py --log-level=info cyhy 2>&1 | /usr/bin/logger -t cyhy-notifications
+    job: cd /var/cyhy/reports && ./create_send_notifications.py --log-level=info cyhy 2>&1 | /usr/bin/logger -t cyhy-notifications
   when: production_workspace
```

Comment:
This should be `production_workspace|bool` to avoid an Ansible 2.8 warning.

Reply:
Good catch, thanks @jsf9k - fixed in 2678ef5